pkg/apis/submariner.io/v1/types.go

```diff
@@ -344,3 +344,51 @@ type GlobalIngressIPList struct {
 
 	Items []GlobalIngressIP `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName="gwrt"
+
+type SubmarinerGWRoute struct {
```

Comment:
I say we drop he `Submariner` prefix. We don't prepend that with our other CRDs.

```suggestion
type GatewayRoute struct {
```